app/controllers/PaymentHistoryController.scala

```diff
@@ -55,7 +55,7 @@ class PaymentHistoryController @Inject()(val paymentHistoryView: PaymentHistory,
                                          val repaymentService: RepaymentService,
                                          val retrieveNinoWithIncomeSources: IncomeSourceDetailsPredicate)
                                         (implicit val appConfig: FrontendAppConfig,
-                                         dateService: DateServiceInterface,
+                                         dateService: DateService,
```

Comment:
this can remain DateServiceInterface,
in case you extending, adding new method etc, you would need to apply changes in all dependencies